sources/packages/web/src/views/institution/locations/confirmation-of-enrollment/COESummaryData.vue

```diff
@@ -6,68 +6,99 @@
           <slot name="coeSummarySubtitle">{{ coeSummarySubtitle }}</slot>
         </template>
         <template #actions>
-          <v-text-field
-            density="compact"
-            label="Search Name"
-            variant="outlined"
-            v-model="searchCriteria"
-            data-cy="searchCriteria"
-            @keyup.enter="searchCOE"
-            prepend-inner-icon="mdi-magnify"
-            hide-details="auto"
-          />
+          <v-row class="justify-end">
+            <v-col cols="auto">
+              <v-btn-toggle
+                v-model="intensityFilter"
+                class="btn-toggle"
+                selected-class="selected-btn-toggle"
+                @update:model-value="resetPageAndLoadEnrollments"
+              >
+                <v-btn
+                  rounded="xl"
+                  color="primary"
+                  :value="IntensityFilter.All"
+                  class="mr-2"
+                  >All</v-btn
+                >
+                <v-btn
+                  v-for="intensity in Object.values(OfferingIntensity)"
+                  :key="intensity"
+                  rounded="xl"
+                  color="primary"
+                  :value="intensity"
+                  class="mr-2"
+                  >{{ mapOfferingIntensity(intensity) }}</v-btn
+                >
+              </v-btn-toggle>
+            </v-col>
+            <v-col cols="3">
+              <v-text-field
+                density="compact"
+                label="Search by name or application number"
+                variant="outlined"
+                v-model="searchQuery"
+                @keyup.enter="resetPageAndLoadEnrollments"
+                prepend-inner-icon="mdi-magnify"
+                hide-details="auto"
+              />
+            </v-col>
+          </v-row>
         </template>
       </body-header>
       <content-group>
-        <toggle-content :toggled="!disbursements.count">
-          <DataTable
-            :value="disbursements.results"
-            :lazy="true"
-            :paginator="true"
-            :rows="pageLimit"
-            :rowsPerPageOptions="rowsPerPageOptions"
-            :totalRecords="disbursements.count"
-            @page="pageEvent"
-            @sort="sortEvent"
+        <toggle-content
+          :toggled="!enrollmentsLoading && !disbursements.count"
+          message="No enrollment records found"
+        >
+          <v-data-table-server
+            v-if="disbursements?.count"
```

Comment:
Was there a need to have the `v-if="disbursements?.count"` in place?
If not, please remove it.